File: Angular/src/app/add-module/education-dialog/domain/institution.ts
export class Institution {
    constructor(private institutionId: string,
        private institutionName: string,
        private startDate: string,
        private endDate: string) {
        this.institutionId = institutionId;
        this.institutionName = institutionName;
        this.startDate = startDate;
        this.endDate = endDate;
    }
}

File: WebsocketService/src/main/java/com/example/WebsocketService/domain/Institution.java
package com.example.WebsocketService.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

//This class is to model the instutition data of educations data
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Institution {

    private String institutionId;
    private String institutionName;
    private String startDate;
    private String endDate;

}


